site/source/tutorials/accessibility.md

```diff
@@ -0,0 +1,233 @@
+
+___
+
+### Dialog
+
+The Dialog widget creates a controlled dialog element. Controlled focus is not yet implemented, but will be added along with the focus manager.
+
+##### A11y properties
+- `closeText`: Provides hidden, accessible text for the icon close button. Defaults to "close dialog."
+- `role`: The default role for the dialog container is `dialog`, but it can be changed to `alertdialog` through this property.
+
+___
+
+### SlidePane
+
+SlidePane is similar to dialog in that it creates a modal that overlays page content. It will need similar programmatic focus control, pending completion of the focus manager.
+
+##### A11y properties
+- `closeText`: Provides hidden, accessible text for the icon close button. Defaults to "close pane."
+
+___
+
+### TabController
+
+TabController creates a set of tabs that control the content of a container. The tabs use `role="tab"` and are associated with the content through `aria-controls` and `aria-labelledby`. Only one tab is in the tab order, and arrow keys are used to switch between tabs.
+
+##### Keyboard Interaction
+- **Left arrow**: Moves to previous tab, wrapping from first to last
+- **Right arrow**: Moves to next tab, wrapping from last to first
+- **Down arrow**: Moves to next tab on vertically oriented tabs
+- **Up arrow**: Moves to previous tab on vertically oriented tabs
+- **Page Up**: Moves to first tab
+- **Page Down**: Moves to last tab
+- **Escape**: Closes focused tab if it is closeable
+
+___
+
+### TitlePane
+
+TitlePane creates an accordion widget, functionally a button that expands a content container. Collapsed content is obscured with `aria-hidden`, and the button and content container are associated with `aria-controls` and `aria-labelledby`.
+
+##### A11y properties
+- `headingLevel`: Optionally customize the heading level of the button controlling the accordion.
+___
+
+This overview only touched on properties that primarily exist for accessibility-related reasons, but any design decision or property change will end up affecting accessibility in some way. For some of those, such as `invalid`, `disabled`, and `readOnly`, we handle setting ARIA attributes in the background in addition to toggling classes without any extra attention needed from the user. In other cases, such as `getResultLabel` in ComboBox or `renderMonthLabel` in Calendar, it is entirely up to the widget user to ensure the returned result includes clear, screen-reader-accessible text content.
+
+## Styling
+The base Dojo 2 theme meets WCAG AA color contrast guidelines, but it is up to the user to double check the final styles as changes to font size or background could affect readability. Widget css packaged with Dojo2 also includes `:focus` styles for all focusable nodes.
+
+There is a set of base styles provided by `@dojo/widgets` separate from themes, containing basic utility classes like the screen reader-accessible visually hidden style. To use it, import `baseCss` separately like so:
```

Comment:
Can we have a sentence that quickly describes what "visually hidden" means so people realize that the difference between that and "display: none;"?